pyfr/plugins/acoustics.py

```diff
@@ -0,0 +1,2106 @@
+# -*- coding: utf-8 -*-
+
+from collections import defaultdict, namedtuple
+from mpi4py import MPI
```

Comment:
Can not safely import anything from mpi4py at the global namespace. Make all sub imports local (get_mpi can help).